governance.md

```diff
@@ -4,46 +4,43 @@ The official version of this document, along with a list of
 individuals and institutions in the roles defined in the governance
 section below, is contained in The Project Governance Repository at:
 
-[https://github.com/jupyter/governance](https://github.com/jupyter/governance)
+[https://github.com/matplotlib/governance](https://github.com/matplotlib/governance)
 
 The Project
 ===========
 
-The Jupyter/IPython Project (The Project) is an open source software project
-affiliated with the 501c3 NumFocus Foundation. The goal of The Project is to
-develop open source software and deploy open and public websites and services
-for reproducible, exploratory and interactive computing. The Software developed
-by The Project is released under the BSD (or similar) open source license,
-developed openly and hosted in public GitHub repositories under the
-[IPython GitHub organization](https://github.com/ipython) and the
-[Jupyter GitHub organization](https://github.com/jupyter). Examples of
-Project Software include the IPython Notebook, the IPython Terminal,
-IPython.parallel, the Jupyter Hub, etc. The Services run by the
-Project consist of public websites and web-services that are hosted
-under the jupyter.org or ipython.org domains. The Services Examples of
-Project Services include the Jupyter and IPython websites
-([http://jupyter.org](http://jupyter.org) and
-[http://ipython.org](http://ipython.org)), nbviewer
-([https://nbviewer.ipython.org](https://nbviewer.ipython.org)) and the
-Jupyter coLaboratory
-([https://colaboratory.jupyter.org](https://colaboratory.jupyter.org)).
+The Matplotlib Project (The Project) is an open source software
+project affiliated with the 501c3 NumFocus Foundation. The goal of The
+Project is to develop open source software and deploy open and public
+websites and services for data visualization. The Software developed
+by The Project is released under the BSD (or similar permissive) open
+source license, developed openly and hosted in public GitHub
+repositories under the
+[Matplotlib GitHub organization](https://github.com/matplotlib). Examples
+of Project Software include the matplotlib library for data
+visualization and associated extensions and dependencies.  The
```

Comment:
"...and associated extensions and dependencies"? what does this mean, specifically? This is really vague.